src/controller/CHIPDevice.cpp

```diff
@@ -164,6 +169,14 @@ CHIP_ERROR Device::Serialize(SerializedDevice & output)
     serializable.mDevicePort = Encoding::LittleEndian::HostSwap16(mDeviceAddress.GetPort());
     serializable.mAdminId    = Encoding::LittleEndian::HostSwap16(mAdminId);
 
+    Transport::PeerConnectionState * connectionState = mSessionManager->GetPeerConnectionState(mSecureSession);
+    VerifyOrExit(connectionState != nullptr, error = CHIP_ERROR_INCORRECT_STATE);
+    localMessageCounter = connectionState->GetSessionMessageCounter().GetLocalMessageCounter().Value();
+    peerMessageCounter  = connectionState->GetSessionMessageCounter().GetPeerMessageCounter().GetCounter();
```

Comment:
peerMessageCounter shouldn't be persistent, it will be negotiated during CASE resumption.

If it is needed temporary to fix CI problem, please add a TODO issue.

Reply:
https://github.com/project-chip/connectedhomeip/pull/7106/files/71468cce3a47939418b33319ad549fd64ede5c9f#r639366772
I think this comment explains why this is needed right now.